docs/troubleshooting/blob-size-limit-error.mdx

```diff
@@ -42,6 +42,6 @@ There are multiple strategies you can use to avoid this error:
      1. Execute Activities in smaller batches within a single Workflow Task.
      2. Introduce brief pauses or sleeps (for example, 1ms) between batches.
 
-3. Consider offloading large payloads to an object store (for example, Amazon S3) to reduce the risk of reaching blob size limits:
+3. Consider offloading large payloads to an object store to reduce the risk of reaching blob size limits:
```

Comment:
Maybe (for example, a cloud-based store)? This edit seems to lose the sense of "cloud" trying to be non-provider specific.

Reply:
I think Object Storage is universal enough that people know what it means, at least in cloud space. I think adding "cloud based store" doesn't add anything.